proto/ha_scope.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package dash.ha_scope;
+
+enum Role{
```

Comment:
HARole might be better.

also is it possible to move the command enums to a shared place?

Reply:
Addressed.